premake5.lua

```diff
@@ -289,13 +289,16 @@
 			include "contrib/curl"
 		end
 
-	group "Binary Modules"
-		include "binmodules/example"
+	filter { "toolset:not cosmocc" }
```

Comment:
Did we change `filter` to work on groups and projects? I didn't think that had happened?